src/HH4b/processors/GenSelection.py

```diff
@@ -64,6 +65,7 @@ def gen_selection_HHbbbb(
     fatjets: FatJetArray,
     selection_args: list,
     skim_vars: dict,
+    fatjet_str: str = "bbFatJet",
```

Comment:
so the options are `bbFatJet` or `ak8FatJet` maybe we can make that explicit somewhere?